AzFilesHybrid/AzFilesHybrid.psm1

```diff
@@ -4029,6 +4113,8 @@ function Join-AzStorageAccount {
     .PARAMETER OverwriteExistingADObject
     The switch to indicate whether to overwrite the existing AD object for the storage account. Default is $false and the script
     will stop if find an existing AD object for the storage account.
+    .PARAMETER EncryptionType

```

Comment:
If customer specifies RC4 and AES and meets the requirements for that, should we consider doing Update-AzStorageAccountSetupForAES256 as part of join-azstorageaccount execution?

Reply:
I thought we are keeping the join cmdlet as is for now and provide an update cmdlet for AES256 if customers want to use AES256. When deprecating RC4, we are going to switch to AES256 in the join cmdlet.